### PR TITLE
[FIX] Change label orientation for upper right quadrant

### LINKED ISSUE #59: Label orientation in plot_connectivity_circle()
#### Describe the bug
I encountered a possible issue with `plot_connectivity_circle()`. When executing `test_plot_connectivity_circle()` with `start_pos=0` instead of `start_pos=90`, the following figure is created:

test

The orientation of labels in the upper right quadrant might not be desired behavior. If you agree with this then a simple fix would be to insert `if angle_deg >= 270 or angle_deg < 90` at this line:

https://github.com/mne-tools/mne-connectivity/blob/f3ff4ed9bb76dd28248f5944bb7abf875ca38b1d/mne_connectivity/viz/circle.py#L319

### PR DESCRIPTION
**Pull request by alexanderkroner**
PR Description
--------------

Closes #59.

Merge checklist
---------------

Maintainer, please confirm the following before merging:

- [ ] All comments resolved
- [ ] This is not your own PR
- [ ] All CIs are happy
- [ ] PR title starts with [MRG]
- [ ] [whats_new.rst](https://github.com/mne-tools/mne-connectivity/blob/main/doc/whats_new.rst) is updated
- [ ] PR description includes phrase "closes <#issue-number>"

### CODE REVIEW

**Comment by adam2392** [comment]
Thanks @alexanderkroner for the PR!

Would you like to add a changelog entry to https://github.com/mne-tools/mne-connectivity/blob/main/doc/whats_new.rst?

File: doc/whats_new.rst [hunk 1]
```diff
@@ -33,6 +33,7 @@ Bug
 
 - Fixed bug when saving connectivity with ``n_jobs`` greater than 1 from :func:`mne_connectivity.spectral_connectivity`, by `Adam Li`_ (:gh:`43`)
 - Fixed bug to allow saving complex data connectivity, by `Adam Li`_ (:gh:`43`)
+- Fixed bug to keep label orientation upright in :func:`mne_connectivity.viz.circle.plot_connectivity_circle`, by `Alexander Kroner`_ (:gh:`60`)
```
Comment (adam2392) [inline doc/whats_new.rst:36]:
```suggestion
- Fixed bug to keep label orientation upright in :func:`mne_connectivity.viz.plot_connectivity_circle`, by `Alexander Kroner`_ (:gh:`60`)
```

**Review by adam2392 — APPROVED** [review]
Good to merge when CIs green.

**Comment by adam2392** [comment]
@larsoner do you know why the circleCI might be failing. I took a look at MNE-Python and the builds are stable and added it to this package, but still not working.

**Comment by larsoner** [comment]
It gets killed during an example, and that usually means that memory consumption is too high. If you're over ~1.5GB for an example, you're going to hit problems. Here is what I did locally:
```
$ mprof run examples/connectivity_classes.py
$ mprof plot
```
Gives a ~3GB peak:

Screenshot from 2021-12-13 16-41-08

If you fix this memory usage, it should fix CircleCI.

**Comment by adam2392** [comment]
Thanks @larsoner that was correct and helpful. I added that to the contributing doc for future ref!

**Comment by adam2392** [comment]
Thanks @alexanderkroner for your PR!

**Pull request merged**